### ISSUE TITLE
Parse error when using generics in template literals

### ISSUE THREAD

**Issue by imm** [DanielXMoore/Civet#819]
Hello everyone,

This is good:
```coffee
{rows} := await sql`SELECT * FROM tbl`
```

This is not:
```coffee
{rows: [user]} := await sql<User>`SELECT * FROM users WHERE id = ${id}`
                                 ^ ParseError
```

Typescript allows generics to be used in template literals.

**Comment by edemaine** [comment]
Thanks for the report! I didn't know TypeScript allowed that. Should be an easy fix.

**Comment by edemaine** [comment]
Should be fixed in 0.6.52. Thanks again!
